usaspending_api/references/abbreviations.py

```diff
@@ -74,6 +74,7 @@
     "PLW": "Palau",
 }
 
+code_to_name = {k: v["name"] for (k, v) in code_to_state.items()}
```

Comment:
Is this necessary? The pre-existing _to_code dictionaries are changing the keys, this is simplifying the values with already present data.

Example derivation function code using the original:
```python
   code_to_state.get(broker_input["legal_entity_state_code"], {}).get("name")
```

Reply:
Doesn't that break if the first .get() returns None? That seemed like a vulnerability in the origional.

Reply:
The default value for the first get is an empty dict, no vulnerability compared to the original.